clang-tools-extra/clang-tidy/bugprone/NarrowingConversionsCheck.cpp

```diff
@@ -260,7 +260,7 @@ static IntegerRange createFromType(const ASTContext &Context,
     llvm::APSInt LowerValue(PrecisionBits + 2, /*isUnsigned*/ false);
     LowerValue.setBit(PrecisionBits);
     LowerValue.setSignBit();
-    return {LowerValue, UpperValue};
+    return {std::move(LowerValue), UpperValue};
```

Comment:
this doesn't have any impact.
And if using move, why not for a second too

Reply:
Isn't this actually an antipattern? What exactly do you mean by "code sanitizer", can you show the actual error message?

Reply:
This is the error message:
" LowerValue is passed-by-value as parameter to llvm::APSInt::APSInt(llvm::APSInt const &) /*implicit =default*/, when it could be moved instead"
"Use std::move(LowerValue) instead of LowerValue."

Reply:
Do you think this should be reverted?

Reply:
What tool did you use to get that error message?

I do not see the point of this change, please revert. We must not blindly do what tools tell us, we must critically think if they make sense or not.